packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -585,7 +588,7 @@ class DatePicker extends React.Component {
         minTime={this.props.minTime}
         maxTime={this.props.maxTime}
         excludeTimes={this.props.excludeTimes}
-        className={this.props.calendarClassName}
+        className={cx(this.props.calendarClassName)}
```

Comment:
We'll want this classname to pass through as-is without it passing through the cx binding that binds the to styles from `'./stylesheets/datepicker.scss'`. We should leave this as it was before since we are taking a prop for this and not one of the classes from our `./stylesheets/datepicker.scss'` file.